sdk/search/azure-search-documents/azure/search/documents/indexes/models/_models.py

```diff
@@ -29,11 +28,10 @@
     SearchIndexerIndexProjections,
 )
 
-
 DELIMITER = "|"
 
 
-class SearchIndexerSkillset(_serialization.Model):
+class SearchIndexerSkillset:
```

Comment:
nit: can also remove the `super().__init__(**kwargs)` in the constructor of this class as well.